cfme/tests/v2v/test_csv_import.py

```diff
@@ -60,10 +61,14 @@ def import_and_check(appliance, infra_map, error_text, filetype='csv', content=F
     except UnexpectedAlertPresentException:
         pass
     if table_hover:
-        wait_for(lambda: plan_view.vms.table.is_displayed,
-                 timeout=60, message='Wait for VMs view', delay=2)
+        wait_for(lambda: plan_view.vms.is_displayed,
+                 timeout=60, message='Wait for VMs view', delay=5)
         if table_hover is 'duplicate':
-            plan_view.vms.table[2][1].widget.click()
+            if appliance.version >= '5.10.0.19':
```

Comment:
I do not think we need this version check. Since we are on 5.10.0.21 now, is testing going to occur on builds before this?

Reply:
@jawatts @digitronik, this version check added with respect to 5.9 basically.
Agree with views https://github.com/ManageIQ/integration_tests/pull/7982#discussion_r228158800, changing it to stream-wise, `5.10` instead of `5.10.0.19`